content/en/tracing/setup_overview/setup/dotnet-framework.md

```diff
@@ -178,6 +178,9 @@ example.exe
 
 <div class="alert alert-warning">
 <strong>Note:</strong> To set environment variables for a Windows Service, use the multi-string key <code>HKLM\System\CurrentControlSet\Services\{service name}\Environment</code> in the Windows Registry, as described above.
+<br />
+<br />
+IIS runs all applications under one Windows Service so it's best to use one of the other configuration options (Code, web.config, JSON file) for application specific settings like DD_SERVICE.
```

Comment:
```suggestion
Because IIS runs all applications under one Windows Service, Datadog recommends you use another configuration option (Code, web.config, or JSON file) for application-specific settings such as <code>DD_SERVICE</code>.
```